packages/modules/create-imodel/src/components/base-imodel/BaseIModel.tsx

```diff
@@ -119,6 +119,15 @@ export function BaseIModelPage(props: BaseIModelProps) {
     ...stringsOverrides,
   };
 
+  React.useEffect(() => {
```

Comment:
Can you write a test so this bug wouldn't happen again?

Reply:
added a test